public/app/core/history/richHistoryLocalStorageUtils.ts

```diff
@@ -25,22 +28,27 @@ export function filterAndSortQueries(
   return sortQueries(filteredQueriesToBeSorted, sortOrder);
 }
 
-export const createRetentionPeriodBoundary = (days: number, isLastTs: boolean) => {
-  const today = new Date();
-  const date = new Date(today.setDate(today.getDate() - days));
+export const createRetentionPeriodBoundary = (
```

Comment:
Mayne this should be moved somewhere near the front end? Not sure so opted for keeping it here for now.